net-irc/emech/emech-3.0.99_p3.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit readme.gentoo-r1 toolchain-funcs
+
+DESCRIPTION="UNIX compatible IRC bot programmed in the C language"
+HOMEPAGE="http://www.energymech.net/"
+SRC_URI="http://www.energymech.net/files/${P/_/}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+IUSE="debug session tcl"
+S="${WORKDIR}/${P/_/}"
+
+DOC_CONTENTS="You can find a compressed sample config file at /usr/share/doc/${PF}"
```

Comment:
Do you really think it's a good idea to create a whole README file for one line of text?